apps/client/src/pages/CheckerPage.js

```diff
@@ -86,6 +89,21 @@ const CheckerPage = ({ checker, topic, resetChecker }) => {
       : console.error(
           `goToQuestion(): ${value} is not an integer, 'next' or 'prev'`
         );
+    const question = checker.stack[questionIndex];
+    // EventName is the question text in lowercase combined with the button that is pressed. (next, prev)
+    const eventName = `${
+      Number.isInteger(value)
+        ? eventNames.EDIT_QUESTION
+        : value === "prev"
+        ? eventNames.PREV_QUESTION
+        : eventNames.NEXT_QUESTION
+    } - ${question.text.toLowerCase()}`;
```

Comment:
I thought we agreed to split it into separate functions for next/prev and gotoindex